lib/metrics.js

```diff
@@ -1,5 +1,5 @@
 var Statsd = require('strong-statsd');
-var agent = require('./agent');
+// var agent = require('./agent');
```

Comment:
I tried that route, things got more complicated since I had to change config.js a lot more.